searchcore/src/apps/vespa-feed-bm/bm_cluster_controller.cpp

```diff
@@ -42,7 +42,8 @@ BmClusterController::BmClusterController(SharedRpcResources& shared_rpc_resource
 void
 BmClusterController::set_cluster_up(bool distributor)
 {
-    StorageMessageAddress storage_address("storage", distributor ? NodeType::DISTRIBUTOR : NodeType::STORAGE, 0);
+    static vespalib::string _storage;
```

Comment:
Shouldn't this be initialised to _"storage"_ ?

Reply:
Yes, it should :)